plugins/lookup/kustomize.py

```diff
@@ -141,7 +155,21 @@ def run(
         if enable_helm:
             command += ["--enable-helm"]
 
-        (ret, out, err) = run_command(command)
+        environ = None
+        if environment:
+            environ = os.environ.copy()
+            if isinstance(environment, str):
```

Comment:
The code here assumes that the value should be defined as `raw`, which can either be a `dict` or a `str` with a specific syntax

Reply:
updated